wecs/panda3d/model.py

```diff
@@ -113,36 +117,39 @@ class PhysicsBody:
 
 
 # Loading / reparenting / destroying models
-class LoadGeometry(System):
+class ManageGeometry(System):
     entity_filters = {
         'model': and_filter([
             Geometry,
             Model,
         ]),
     }
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.cardmaker = CardMaker("maker of cards")
-        # set frame so the bottom edge is centered on 0
-        self.cardmaker.set_frame(-0.5,0.5,0,1)
-
     def init_entity(self, filter_name, entity):
         geometry = entity[Geometry]
```

Comment:
Even if there is currently only one filter, please check for its name before running code, otherwise if someone patches the system and adds another filter, this code may also be run if the developer forgot to go over this code's semantics again. Defensive programming.